docs/csharp/asynchronous-programming/snippets/generate-consume-asynchronous-streams/finished/Program.cs

```diff
@@ -100,7 +100,7 @@ await foreach (var issue in RunPagedQueryAsync(client, PagedIssueQuery, "docs")
         // Stop with 10 pages, because these are large repos:
         while (hasMorePages && (pagesReturned++ < 10))
         {
-            var postBody = issueAndPRQuery.ToJsonText();
+            string postBody = JsonConvert.SerializeObject(issueAndPRQuery); // Serialize to JSON string
```

Comment:
This is the same code as what the `ToJsonText` function does, how does this solve anything related to the issue in question?